### ISSUE TITLE
Anyone getting errors while deploying Machine Learning after putting a thumbprint

### ISSUE THREAD

**Issue by iomolo** [Azure-Samples/MyDriving#606]

**Comment by icsimlai** [comment]
Yes, I am getting the same error. Did you find any work around ?

**Comment by icsimlai** [comment]
The issue is in the below line "$res = InvokeManagementAPI $subscriptionId "cloudservices/$workspaceName/resources/machinelearning/~/workspaces" $thumbprint" in "copyMLExperiment.ps1"
Could someone please let me know what is the correct value for below parameter ? , "cloudservices/$workspaceName/resources/machinelearning/~/workspaces"

**Comment by icsimlai** [comment]
The issue is in the below line,
 $res = InvokeManagementAPI $subscriptionId "cloudservices/$workspaceName/resources/machinelearning//workspaces"
 $thumbprint" 
in "copyMLExperiment.ps1" file.
Could someone please let me know what is the correct value for below parameter ? , "cloudservices/$workspaceName/resources/machinelearning//workspaces"

**Comment by iomolo** [comment]
Were you able to get any solution on this? Am still all over the place to get this working

**Comment by icsimlai** [comment]
Not yet ...
